gce_eligibility_hexes.sql

```diff
@@ -0,0 +1,290 @@
+CREATE MATERIALIZED VIEW gce_eligibility_hexes AS (
```

Comment:
this file surely belongs somewhere more connected to nycdb, but I've left it here for now.